UI/web-src/ngMango/components/userTable/userTable.html

```diff
@@ -1,18 +1,24 @@
 <div flex="noshrink" layout="column" ng-form="userTable" ma-form-exclude>
+
     <div class="ma-virtual-repeat-table">
         <div class="ma-virtual-repeat-table-header">
             <div class="ma-virtual-repeat-table-row">
                 <div class="ma-checkbox-column">
                     <md-menu-bar>
                         <md-menu md-offset="-4 4">
-                            <md-button class="md-icon-button ma-small-button ma-no-margin" ng-click="$ctrl.openMenu($event, $mdMenu)" tabindex="2">
+                            <md-button class="md-icon-button ma-small-button ma-no-margin"
+                                ng-click="$ctrl.openMenu($event, $mdMenu)" tabindex="2">
                                 <md-icon md-menu-origin>more_vert</md-icon>
                                 <md-tooltip><span ma-tr="ui.app.moreOptions"></span></md-tooltip>
                             </md-button>
-                            <md-menu-content width="3" class="ma-user-table" md-colors="::{background: 'background-hue-2'}">
+                            <md-menu-content width="3" class="ma-user-table"
+                                md-colors="::{background: 'background-hue-2'}">
                                 <md-menu-item ng-if-start="$ctrl.selectMultiple">
                                     <md-button ng-disabled="true">
-                                        <span ma-tr="ui.app.xItemsSelectedLong" ma-tr-args="[$ctrl.selectedItems.size]"></span>
+                                        <span ma-tr="ui.app.xItemsSelectedLong"
+                                            ma-tr-args="[$ctrl.selectedItems.size]"></span>
+                                        <span>&nbsp;</span>
+                                        <span ma-tr="ui.app.outOf" ma-tr-args="[$ctrl.pages.$total]"></span>
```

Comment:
Please avoid using translations for incomplete sentences. You should add a new translation for the complete sentence and use a single span.